0.10.4/alpine/Dockerfile

```diff
@@ -0,0 +1,77 @@
+FROM alpine:latest
+
+MAINTAINER João Fonseca <jfonseca@uphold.com>
+
+RUN adduser -S litecoin
+
+ENV GOSU_VERSION=1.7 \
+  GOSU_SHASUM="34049cfc713e8b74b90d6de49690fa601dc040021980812b2f1f691534be8a50  /usr/local/bin/gosu"
+
+RUN apk --no-cache add openssl \
```

Comment:
We should normalize the installation of gosu between our clients: [bitcoin](https://github.com/seegno/docker-bitcoind/blob/master/0.12/Dockerfile#L16), etc.